File: frontend/chessandconnect/src/app/models/WebSocketMessages/GameInvitationModel.ts
import { FriendshipState } from "../../enums/FriendshipState";
import { GameType } from "../../enums/game";

export class GameInvitationModel {
    HostId!: number;
    FriendId!: number;
    State!: FriendshipState;
    Game!: GameType
}

File: backend/chess4connect/Models/SocketComunication/MessageTypes/GameInvitationModel.cs
using chess4connect.Enums;

namespace chess4connect.Models.SocketComunication.MessageTypes;

public class GameInvitationModel
{
    public int HostId { get; set; }
    public int FriendId { get; set; }
    public FriendshipState State { get; set; }
    public GameType Game { get; set; }
}
